runtime-testsuite/test/org/antlr/v4/runtime/TestCodePointCharStream.java

```diff
@@ -3,17 +3,13 @@
  * Use of this file is governed by the BSD 3-clause license that
  * can be found in the LICENSE.txt file in the project root.
  */
-package org.antlr.v4.test.runtime.java;
+package org.antlr.v4.runtime;
```

Comment:
Ah, I see the confusion. I didn't move which Maven target it was in, so it won't show up in the runtime jar. I just put it in the same Java package so it could access package-private methods.